controllers/tempo/suite_test.go

```diff
@@ -26,6 +26,7 @@ func TestMain(m *testing.M) {
 		//	Paths: []string{filepath.Join("..", "config", "webhook")},
 		//},
 	}
+	testEnv.BinaryAssetsDirectory = "/home/rvargasp/.local/share/kubebuilder-envtest/k8s/1.24.2-linux-amd64/"
```

Comment:
This should not be needed since `KUBEBUILDER_ASSETS` is set.

Reply:
Done, thanks!

Reply:
It's required when you run the test from the "Testing" tab in VS Code. I'd also like this fixed somehow, but of course we don't want to hardcode this absolute path (including a version) :|

Reply:
Yes  exactly, I tried to put in my `launch.json` the env variable `KUBEBUILDER_ASSETS` and it didn't work, so this was an easy "fix". :D